test/functional/ex_cmds/ctrl_c_spec.lua

```diff
@@ -16,7 +16,7 @@ describe("CTRL-C (mapped)", function()
     end
 
     source([[
-      set nomore nohlsearch undolevels=-1
```

Comment:
I'd feel better if we left the explicit cases as they were. At least for one reason: we may want to revert the global change later. With https://github.com/neovim/neovim/pull/6247 and/or an enhancement to the test system, we may be able to auto-detect hung tests.